From 6ab441e0509a9fda179fd1febf73d0f9fae321d6 Mon Sep 17 00:00:00 2001
From: Samuel Martin <s.martin49@gmail.com>
Date: Fri, 2 May 2014 14:24:58 +0200
Subject: [PATCH] cmake: set the project language

By default, CMake sets C and C++ as languages used by the project.
This makes the configuration failed if no C++ compiler is found.

Since libwesockets in written in pure C, no C++, there is no reason to
disable it when the toolchain has no C++ support.

This patch just make the CMake code reflecting the real status of the
project with regard to the languages used.

Signed-off-by: Samuel Martin <s.martin49@gmail.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f1b6ace..72391b0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 2.6)
 
-project(libwebsockets)
+project(libwebsockets C)
 
 set(PACKAGE "libwebsockets")
 set(CPACK_PACKAGE_NAME "${PACKAGE}")
-- 
1.9.2

